src/app/forms/components/defect-select/defect-select.component.ts

```diff
@@ -12,7 +12,7 @@ import { Subject } from 'rxjs';
 export class DefectSelectComponent implements OnDestroy {
   @Input() defects!: Defect[] | null;
   @Input() isParentEditing = false;
-  @Output() formChange = new EventEmitter<{ defect: Defect, item: Item, deficiency: Deficiency }>();
+  @Output() formChange = new EventEmitter<{ defect: Defect, item: Item, deficiency?: Deficiency }>();
 
   isEditing = false;
   selectedDefect?: Defect;
```

Comment:
There's a rogue `destroy$` in the component

Reply:
Removed.